$NetBSD: patch-kdc_version-script.map,v 1.2 2015/07/11 04:26:57 rodent Exp $

Adds the symbols _kdc_db_fetch and _kdc_free_ent to global
visibility, so that they can be referenced from kdc/digest-service.
Fixes build on Dragonfly. From Alex Hornung in PR pkg/45195.

--- kdc/version-script.map.orig	2012-12-09 22:06:44.000000000 +0000
+++ kdc/version-script.map
@@ -2,6 +2,8 @@
 
 HEIMDAL_KDC_1.0 {
 	global:
+		_kdc_db_fetch;
+		_kdc_free_ent;
 		kdc_log;
 		kdc_log_msg;
 		kdc_log_msg_va;
